includes/discovery/functions.inc.php

```diff
@@ -449,6 +450,9 @@ function sensor_limit($class, $current)
         case 'snr':
         case 'frequency':
         case 'pressure':
+        case 'probes':
+            $limit = 5;
```

Comment:
This changes the limit for airflow, Snr, frequency and pressure also, I'm guessing this is not intended?

Reply:
Yes, good point. Will do.

Reply:
Fixed in latest commit, thanks :+1: 

PelliX